.github/workflows/msvc-ci.yml

```
@@ -0,0 +1,89 @@
+---
+name: MSVC build and test
+on:
+  push:
+    branches:
+      - 'master'
+  pull_request:
+    branches:
+      - '*'
+  release:
+
+jobs:
+  test:
+    name: Run tests
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        include: [
+          { os: windows-2019, tests: none},
+          { os: windows-2019, tests: main},
+          { os: windows-2016, tests: main},
+          { os: windows-2016, tests: none},
+        ]
+    steps:
+      - name: Checkout source
+        uses: actions/checkout@v2
+        with:
+          path: ${{ github.event.repository.name }}
+      - name: Fetch Data-Sample repository
+        uses: actions/checkout@v2
+        with:
+          repository: coin-or-tools/Data-Sample
+          path: Data/Sample
+      - name: Fetch ThirdParty-Glpk repository
+        uses: actions/checkout@v2
+        with:
+          repository: coin-or-tools/ThirdParty-Glpk
+          path: ThirdParty/Glpk
+      - name: Checkout coinbrew
+        uses: actions/checkout@v2
+        with:
+          repository: coin-or/coinbrew
+          path: coinbrew
+      - name: Set up msvc
+        uses: ilammy/msvc-dev-cmd@v1
+      - name: Set up msys
+        uses: msys2/setup-msys2@v2
+        with:
+          update: true
+          install: >-
+            base-devel
+            git
+          path-type: inherit
+      - name: Build project
+        run: |
+          [[ ${{ matrix.tests }} == "none" ]] && debug="" suffix="" || debug="--enable-debug" suffix="-dbg"
```

Would it be nicer to encode the MSVC runtime variant (static vs dynamic, debug or not) explicitly in the suffix?
That is,
```suggestion
          [[ ${{ matrix.tests }} == "none" ]] && debug="" suffix="-md" || debug="--enable-debug" suffix="-mdd"
```
Would make it easier to users to see how these libs have been build.
Also, some day someone will probably come to request builds with `-MT/-MTD` and Ted may give in and want to add these.

---

So far, we're just using the default run-time and haven't added run-time variants into the matrix as we had in appveyor. I guess we should add the naming only after adding different variants to the matrix. @xmunoz do you prefer to do that here or in a new PR?

---

Well, OK, thinking about it further, we should probably just do this now. Did we decide to abandon the -MT builds in Appveyor, I can't remember.

---

Yeah, not sure. Should I commit @svigerske's suggestion?

---

Yeah, let's do it, there's more we'll need to think about regarding this, but we do want the run-time encoded into the name for sure in the long-run, since it is necessary to know that for library compatibility as a user.